qiskit/transpiler/preset_passmanagers/plugin.py

```diff
@@ -161,10 +161,11 @@ def pass_manager(self, pass_manager_config, optimization_level):
    PassManagerStagePlugin
    PassManagerStagePluginManager
    list_stage_plugins
+   entry_point_obj
```

Comment:
I think the docs failure is just because this line needs updating with the new name.

Reply:
yeap.. forgot that. Fixed in 6135d9f6c846dcc9f0671ef5a1a1971166975011